clients/nodejs/PlatformPromiseClient.js

```diff
@@ -273,9 +276,17 @@ class PlatformPromiseClient {
             ),
           ),
         ],
+        ...options,
       },
     );
   }
+
+  /**
+   * @param {string} protocolVersion
+   */
+  setProtocolVersion(protocolVersion) {
```

Comment:
Do you want to bring it back?

Reply:
@shumkov I just moved it to the end of the class :)